src/sparseml/modifiers/quantization/pytorch.py

```diff
@@ -131,26 +129,29 @@ def _enable_module_qat(self, module: Module):
         module.apply(torch.quantization.enable_observer)
 
         if not self.qat_enabled_:
-            # fuse conv-bn-relu blocks prior to quantization emulation
-            self._fuse(module)
-
-            # add quantization_schemes to target submodules
-            set_quantization_schemes(
-                module,
-                scheme=self.scheme,
-                scheme_overrides=self.scheme_overrides,
-                ignore=self.ignore,
-                strict=self.strict,
-            )
+            from torch.distributed.fsdp import FullyShardedDataParallel
+
+            with FullyShardedDataParallel.summon_full_params(module):
```

Comment:
need to make this safe for pytorch < 2

maybe have try/except to assign a context manager with except as nullcontext